backend/src/CLI.ts

```diff
@@ -146,14 +146,25 @@ export default class CLI {
 
   async isNotInstalled() {
     const installed = RemoteitInstaller.fileExists(RemoteitInstaller.binaryName)
-    Logger.info('CLI INSTALLED?', { installed, name: RemoteitInstaller.binaryName })
+    d('CLI INSTALLED?', { installed, name: RemoteitInstaller.binaryName })
```

Comment:
is d a shortcut for debugger?

Reply:
Yes, it is a debugger method that is conditional based on how you start the app.